shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java

```diff
@@ -31,10 +31,16 @@
 import tachyon.job.JobConf;
 
 /**
- * TODO(yupeng): add javadoc
+ * Creates a new lineage for given input and output file names with optional exclusion parameters.
```

Comment:
Creates a lineage for the given input files, output files, and command line job.